.github/workflows/prettier.yaml

```diff
@@ -14,23 +17,38 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
         with:
-          ref: ${{github.event.pull_request.head.ref}}
-          repository: ${{github.event.pull_request.head.repo.full_name}}
-          fetch-depth: 0
-
-      - name: Check if PR is from a fork
-        id: check_fork
-        run: echo "::set-output name=is_fork::${{ github.event.pull_request.head.repo.fork }}"
+          fetch-depth: 0  # Needed to compare with origin/main
 
-      - name: Prettify code
-        id: prettify
-        uses: creyD/prettier_action@v4.6
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
         with:
-          prettier_options: --write **/index.html --print-width 200
-          only_changed: True
-          commit_message: "chore: prettier"
-        continue-on-error: true
-
-      - name: Handle commit failure
-        if: ${{ steps.prettify.outcome == 'failure' && steps.check_fork.outputs.is_fork == 'true' }}
-        run: echo "Please run prettier locally"
+          node-version: 'lts/*'
+
+      - name: Install Prettier
+        run: npm install --global prettier
```

Comment:
I would specify a version here. Otherwise there's a chance the two PRs using different versions get merge conflicts.